docs/joining/application.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Applying as OpenRail Project
+weight: 100
+---
+
+# Applying to become an OpenRail project
+
+The OpenRail Association provides hosting to open source projects within the railway sector. Its goal is to facilitate collaboration across organizations so that the huge challenges of the sector can be addressed in a joint effort, effectively and efficiently. The OpenRail Association is open to all participants of the sector, railway companies, vendors, and research institutes, which commit to working on open source software in an open model.
```

Comment:
Again, "Open Source". Further occurrences are below, a global search/replace would make most sense probably.

```suggestion
The OpenRail Association provides hosting to Open Source projects within the railway sector. Its goal is to facilitate collaboration across organizations so that the huge challenges of the sector can be addressed in a joint effort, effectively and efficiently. The OpenRail Association is open to all participants of the sector, railway companies, vendors, and research institutes, which commit to working on Open Source software in an open model.
```